portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsTemplate.testcase

```diff
@@ -216,6 +216,106 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test UseDefaultTemplateWhenCreatingNewPublication {
+		task ("Add new user with publications permitions") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "userea@liferay.com");
+
+			JSONRole.addRegularRole(
+				roleKey = "Publications Role",
+				roleTitle = "Publications Role");
+
+			for (var resourceActionIndex : list "ACCESS_IN_CONTROL_PANEL,ADD_TO_PAGE,CONFIGURATION,PERMISSIONS,PREFERENCES,VIEW") {
+				Permissions.definePermissionViaJSONAPI(
+					resourceAction = ${resourceActionIndex},
+					resourceName = "com_liferay_change_tracking_web_portlet_PublicationsPortlet",
+					roleTitle = "Publications Role");
+			}
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "VIEW_CONTROL_PANEL",
+				resourceName = 90,
+				roleTitle = "Publications Role");
+
+			for (var resourceActionIndex : list "ADD_PUBLICATION,PERMISSIONS") {
+				Permissions.definePermissionViaJSONAPI(
+					resourceAction = ${resourceActionIndex},
+					resourceName = "com.liferay.change.tracking",
+					roleTitle = "Publications Role");
+			}
+
+			for (var resourceActionIndex : list "DELETE,PERMISSIONS,PUBLISH,UPDATE,VIEW") {
+				Permissions.definePermissionViaJSONAPI(
+					resourceAction = ${resourceActionIndex},
+					resourceName = "com.liferay.change.tracking.model.CTCollection",
+					roleTitle = "Publications Role");
+			}
+
+			JSONRole.assignRoleToUser(
+				roleTitle = "Publications Role",
+				userEmailAddress = "userea@liferay.com");
+		}
```

Comment:
Instead of creating your own role from scratch, try using the default "Publications User" role. See `PublicationsUserCollaboration#InviteUserAsEditor` for example.